tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/BREAKING_CHANGES.markdown

```diff
@@ -0,0 +1,537 @@
+
+#### Why was this change made?
+
+This change was made to avoid using an old proprietary solution (NTLM). Kerberos
+is now recommended, which is a standard protocol and a more secure method of
+authentication compared to NTLM.
+
+---------------------------------------
+
+### Deprecated OpenID in Portal Distribution
+- **Date:** 2019-Jan-21
+- **JIRA Ticket:** [LPS-88906](https://issues.liferay.com/browse/LPS-88906)
+
+#### What changed?
+
+OpenID modules have been moved to a new project named `portal-security-sso-openid`.
+This new project is deprecated and available to download from Liferay Marketplace.
+
+#### Who is affected?
+
+This affects anyone using OpenID as an authentication system.
+
+#### How should I update my code?
+
+If you want to continue using OpenID as an authentication system, you must
+download the corresponding module from Liferay Marketplace. Alternatively, you
+should migrate to OpenID Connect, available on Liferay Portal Distribution.
+
+#### Why was this change made?
+
+This change was made to avoid using a deprecated solution (OpenID). OpenID
+Connect is now recommended, which is a more secure method of authentication
+since it runs on top of OAuth.
+
+---------------------------------------
+
+### Deprecated Google SSO in Portal Distribution
+- **Date:** 2019-Jan-21
+- **JIRA Ticket:** [LPS-88905](https://issues.liferay.com/browse/LPS-88905)
+
+#### What changed?
+
+Google SSO modules have been moved from the `portal-security-sso` project
+to a new project named `portal-security-sso-google`.
+This new projects is deprecated and available to download from Liferay Marketplace.
+
+#### Who is affected?
+
+This affects anyone using Google SSO as an authentication system.
+
+#### How should I update my code?
+
+If you want to continue using Google SSO as an authentication system, you must
+download the corresponding module from Liferay Marketplace. Alternatively, you
+can use OpenID Connect.
+
+#### Why was this change made?
+
+This change was made to avoid using an old solution for authentication (Google
+SSO). OpenID Connect is the recommended specification to use Google
+implementation for authentication.
+
+---------------------------------------
+
+### Deprecated dl.tabs.visible property
+- **Date:** 2019-Apr-10
+- **JIRA Ticket:** [LPS-93948](https://issues.liferay.com/browse/LPS-93948)
+
+#### What changed?
+
+The `dl.tabs.visible` property let users toggle the visibility of the navigation
+tabs in Documents & Media when put on a widget page.  This configuration option
+has been removed, so that the navigation tab will never be shown in widget pages.
+
+#### Who is affected?
+
+This affects anyone with a value for `dl.tabs.visible` different than `false`.
+
+#### How should I update my code?
+
+No code changes are necessary.
+
+#### Why was this change made?
+
+Documents & Media has been reviewed from an UX perspective, and removing the
+navigation tabs in widget pages was part of a UI clean up process.
```

Comment:
@Seiphon  can you create issues on github for all of the breaking changes checkers that we need to write to cover the 7.2 breaking changes?

Reply:
@jtydhr88 you can target them to the m3 milestone

Reply:
sure, will do that